core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -118,6 +118,26 @@ export class PickerColumnInternal implements ComponentInterface {
     }
   }
 
+  componentDidRender() {
+    const { activeItem, el, items, isColumnVisible, value } = this;
+
+    if (isColumnVisible) {
+      if (activeItem) {
+        if (el.scrollTop !== this.getScrollTopPositionForElement(activeItem)) {
+          // Scroll the active item into the view if after rendering
+          // it is still not visible
+          this.scrollActiveItemIntoView();
+        }
+      } else if (items[0].value !== value) {
```

Comment:
Need to account for the empty case where `items` is `[]`

```suggestion
      } else if (items[0]?.value !== value) {
```